docs/build/group-chat.md

```diff
@@ -696,11 +706,15 @@ stream.stop()
 
 ## Sync group chats
 
-Calling `sync()` for a group or groups gets any updates since the last sync and adds them to the local database. Be sure to periodically synchronize each group chat to ensure your app has the latest group chat details, including the most recent messages, member list, and group chat details, for example.
+Calling `sync()` for a group or groups gets any updates since the last sync and adds them to the local database.
+
+Sync group chats anytime a user accesses a conversation view. This sync is especially important if a user backgrounds (switches away from) your app, then reopens the app and accesses a conversation view. 
```

Comment:
When we say "conversation view" - just to be sure - Is this the view when a user opens a group chat and can see the messages in the group chat?

Reply:
Yes exactly.